internal/pkg/externalplugins/cherrypicker/cherrypicker_invite_test.go

```diff
@@ -0,0 +1,200 @@
+			commentUser: patternErrWhenCreateComment,
+			wantError:   true,
+			isMember:    true,
+			wantHad:     true,
+		},
+		{
+			name:        "invite when he is not a collabtor, error when judge member",
+			commentUser: patternErrWhenJudgeMember,
+			wantError:   true,
+			isMember:    false,
+			wantHad:     false,
+		},
+		{
+			name:        "invite when he is not a member of our ORG",
+			isMember:    false,
+			commentUser: "wiseguy",
+			wantError:   false,
+			wantHad:     false,
+		},
+		{
+			name:        "invite when he is not a member of our ORG, error when comment",
+			isMember:    false,
+			commentUser: patternErrWhenCreateComment,
+			wantError:   true,
+			wantHad:     false,
+		},
+		{
+			name:          "invite error when judge",
+			collaborators: nil,
+			commentUser:   patternErrWhenIsCollaborator,
+			wantError:     true,
+			wantHad:       false,
+		},
+		{
+			name:          "invite error when add",
+			collaborators: nil,
+			isMember:      true,
+			commentUser:   patternErrWhenAddCollaborator,
+			wantError:     true,
+			wantHad:       false,
+		},
+		{
+			name:          "invite error when add, and comment failed",
+			collaborators: nil,
+			isMember:      true,
+			commentUser:   patternErrWhenAddCollaborator + patternErrWhenCreateComment,
+			wantError:     true,
+			wantHad:       false,
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			ghc := &fghc{
+				pr: &github.PullRequest{
+					Base:      github.PullRequestBranch{Ref: "master"},
+					Number:    2,
+					Merged:    true,
+					Title:     "This is a fix for X",
+					Body:      body,
+					Assignees: []github.User{{Login: "user2"}},
+				},
+				isMember:      tt.isMember,
+				members:       tt.members,
+				patch:         patch,
+				collaborators: tt.collaborators,
+			}
+			ic := github.IssueCommentEvent{
+				Action: github.IssueCommentActionCreated,
+				Repo: github.Repo{
+					Owner:    github.User{Login: "foo"},
+					Name:     "bar",
+					FullName: "foo/bar",
+				},
+				Issue: github.Issue{Number: 2, State: "closed", PullRequest: &struct{}{}},
+				Comment: github.IssueComment{
+					User: github.User{Login: tt.commentUser},
+					Body: "/cherry-pick-invite",
+				},
+			}
+			s := &Server{
+				BotUser:                botUser,
+				GitClient:              c,
+				ConfigAgent:            ca,
+				Push:                   func(forkName, newBranch string, force bool) error { return nil },
+				GitHubClient:           ghc,
+				WebhookSecretGenerator: getSecret,
+				GitHubTokenGenerator:   getGithubToken,
+				Log:                    logrus.StandardLogger().WithField("client", "cherrypicker"),
+				Repos:                  []github.Repo{{Fork: true, FullName: "ci-robot/bar"}},
+			}
+
+			if err := s.handleIssueComment(logrus.NewEntry(logrus.StandardLogger()),
+				&ic); (err != nil) != tt.wantError {
+				t.Fatalf("got error: %v, expected error: %v", err, tt.wantError)
```

Comment:
wantError is a boolean variable, not an error

Reply:
it's right: 
```go
(err != nil) != tt.wantError
```

Reply:
I means, the expected error and got error in the log output.

```
t.Fatalf("got error: %v, expected error: %v", err, tt.wantError)
```